### DISCUSSION TITLE
3.7.0 Pre-Release

### DISCUSSION THREAD

**Discussion by snowman2** [pyproj4/pyproj#1443] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/pyproj4/pyproj/releases/tag/3.7.0rc0'>3.7.0rc0</a>.</em><hr />

Release Candidate 0 Status:

- [x] Linux ARM wheels (https://github.com/pyproj4/pyproj-wheels/pull/59)
- [ ] Conda Forge
- [x] Debian (@sebastic)
- [x] Fedora (@jdkloe)

Plan to release soon due to Python 3.13 release on October 1st. Testing is always appreciated, but definitely not required.

**Comment by sebastic** [comment]
The Debian package built successfully on all release architectures:

 https://buildd.debian.org/status/package.php?p=python-pyproj&suite=experimental

**Reply by snowman2** [reply]
Thanks!

**Comment by jdkloe** [comment]
This version also looks good on Fedora.
The package compiles, tests and builds successfully on Fedora 39, 40, 41, and rawhide (to be v42). 
(only had time to do local tests on x86_64 architecture).

**Reply by snowman2** [reply]
Thanks @jdkloe 👍
